src/Http/Controllers/ImagesController.php

```diff
@@ -24,16 +26,21 @@ public function __construct(
         $this->imagesService = $imagesService;
     }
 
-    public function image(Request $request) : RedirectResponse
+    public function image(Request $request): RedirectResponse
     {
         $path = $request->get('path');
         $params = $request->except(['path']);
 
-        $output = $this->imagesService->render($path, $params);
-        return redirect($output['url']);
+        if (RateLimiter::retriesLeft('resize-image:' . $path, $perMinute = 5)) {
```

Comment:
https://laravel.com/api/8.x/Illuminate/Cache/RateLimiter.html#method_retriesLeft latter says `maxAttempts` nothing to do with `minutes`

Reply:
It depends on decay time, and by default it is a minute (60 seconds)